docs/source/examples.rst

```diff
@@ -6,6 +6,19 @@ Usage Examples
 .. toctree::
    :maxdepth: 1
 
+.. _example_dependencies:
+Example Dependencies
+------------------------
+
+Some examples may use additional dependecies. These dependencies are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
+Pip should be used to install these extra requirements except astra which should be installed via conda:
```

Comment:
I suggest a minor rephrasing to:
The additional requirements should be installed via `pip` [or perhaps ``pip``?], with the exception of `astra` [or `astra-toolbox`?], which should be installed via `conda`:

Reply:
Fixed it.